scripts/deploy-local.js

```diff
@@ -6,7 +6,11 @@ const { prepareToucanEnv } = require("../lib/toucan");
 async function deploy() {
 	const [owner] = await hre.ethers.getSigners();
 	const regenBridgeAccount = "0x11241e35B3f79099123aA0C1C4c97b1FcdCd21f6";
-	const [registry, tco2Factory, tco2] = await prepareToucanEnv(owner, owner);
+	const [registry, tco2Factory, data] = await prepareToucanEnv(owner, owner);
+	let dataAddresses = {};
+	Object.keys(data).forEach(function (key) {
```

Comment:
fyi: there's a more readable alternative for that:
`for (const key in data) {}`

Reply:
Done!